src/server/pps/server/api_server.go

```diff
@@ -996,7 +1034,7 @@ func (a *apiServer) pipelineManager(ctx context.Context, pipelineInfo *pps.Pipel
 		}
 
 		// Start worker pool
-		a.workerPool(ctx, PipelineRcName(pipelineInfo.Pipeline.Name, pipelineInfo.Version))
+		a.workerPool(ctx, PipelineRcName(pipelineInfo.Pipeline.Name, pipelineInfo.Version), RetriesPerDatum)
```

Comment:
It could, the only reason I passed it through is because I'm imagining that we'll want to make it a configurable part of pipelines in the near term future.